tests/Activations/EloquentActivationTest.php

```diff
@@ -26,31 +26,43 @@
 
 class EloquentActivationTest extends TestCase
 {
+    /**
+     * The Activation Eloquent instance.
+     *
+     * @var \Cartalyst\Sentinel\Activations\EloquentActivation
+     */
+    protected $activation;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp(): void
+    {
+        $this->activation = new EloquentActivation();
+    }
+
     /**
      * {@inheritdoc}
      */
     protected function tearDown(): void
     {
+        $this->activation = null;
```

Comment:
Just a line break separating the code

```suggestion
        $this->activation = null;

```